doc/1/controllers/index/refresh/snippets/refresh.cs

```diff
@@ -0,0 +1,2 @@
+await kuzzle.Index.RefreshAsync("nyc-open-data");
+Console.WriteLine("0 shards fail to refresh");
```

Comment:
```suggestion
Console.WriteLine("0 shards failed to refresh");
```

Also, I find the following a tad simpler:
```suggestion
Console.WriteLine("All shards refreshed the index");
```